papers/haskell-exchange-2019/stg-grin/stg-grin.note

```diff
@@ -0,0 +1,25 @@
+- STG in GHC pipeline (rewrite rules, core plugins, core opts before STG, core has types, STG has none)
```

Comment:
I think this shouldn't be part of the PR. :)

Reply:
Done.